client/src/components/scans/scanHostList.js

```diff
@@ -6,9 +6,9 @@ import { EmptyState, Grid, Icon, Modal } from 'patternfly-react';
 
 class ScansHostList extends React.Component {
   render() {
```

Comment:
Note:  Sweet, and another we could embrace a little bit more on the stateless component side with this one, next round